### PR TITLE
Color Scheme logos

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description

Dynamically changes logos according to the user's prefers-color-scheme

## Type of change

- Feature

image
image

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -32,10 +32,10 @@ let octokit: any;
 export async function activate(context: vscode.ExtensionContext) {
   setLoggedIn(false);
   var extensionPath = Path.join(context.extensionPath, "package.json");
-  var packageFile = JSON.parse(fs.readFileSync(extensionPath, 'utf8'));
+  var packageFile = JSON.parse(fs.readFileSync(extensionPath, "utf8"));
```
Comment (EstebanDalelR) [inline src/extension.ts:35]: Formatting only

**Review by baristaGeek — CHANGES_REQUESTED** [review]
Nice, but logo looks huge. Please resize

**Comment by EstebanDalelR** [comment]
> Nice, but logo looks huge. Please resize

There were no changes in it's CSS. Care to provide screenshots?

**Comment by baristaGeek** [comment]
Before this PR
Screen Shot 2022-05-03 at 9 49 38 PM

After this PR
Screen Shot 2022-05-03 at 9 50 01 PM

**Comment by EstebanDalelR** [comment]
I've set it to the same 300px @baristaGeek

**Pull request merged**